drv/cosmo-seq-server/src/vcore.rs

```diff
@@ -377,12 +377,16 @@ impl VCore {
         };
 
         let ereport = Ereport {
-            k: "pmbus.alert",
+            k: if !power_good {
+                "hw.pwr_good.bad"
+            } else {
+                "hw.pmbus.alert"
+            },
```

Comment:
Do we want to change the class for pmbus alerts? If we find power isn't good, is that something we should generate a separate ereport for?

Reply:
I guess the distinction here is that we're getting this by polling PMBus alert and noticing it. If we were actually paying attention to the PG outputs and acting on that, then I think the separate power good class would feel like it made more sense to me. In this case maybe we should be asking what would actually notice this and tell us and the fact that if we lost power on one rail we're losing it on several, but the PMBus alert as to why may be what's distinctively useful here.

I guess to summarize, since we're getting this by polling alert, probably a consistent class makes the most sense?